### ISSUE TITLE
Missing 'import "fmt"' to make ExampleAdd() work in Integers/Examples

### ISSUE THREAD

**Issue by markzzzsmith** [quii/learn-go-with-tests#173]
Text in the book says to add the ExampleAdd() function to adder_test.go, however it won't compile because the "fmt" package hasn't been imported into adder_test.go

Easy and simple to fix, perhaps instead it could be an opportunity to suggest to the reader to have a go at fixing the problem before telling them what the fix is.

**Comment by quii** [comment]
@markzzzsmith How about this? https://github.com/quii/learn-go-with-tests/commit/dde67d3405a4c80e7330128e7bc151a97ca2b77a

**Comment by markzzzsmith** [comment]
Hmm. Admittedly I'm using Vi through habit, and like typing in all the code
from the text rather than just downloading it, because typing it all
assists with learning.

Many years ago I learnt COBOL in a programming class. There is a lot of
templated text in COBOL (IDENTIFICATION SECTION, ENVIRONMENT SECTION etc.),
so I just started importing that text via prepared templates.

I was getting A grades for all the assignments. However, when it came to
the exam, I did quite badly, because I couldn't use the templates. So I'm a
bit wary of tools and techniques that can make learning a programming
language "easier" than it should be.
